### PR TITLE
docs: updated _Working with Compose_

### LINKED ISSUE #3557: Docs: Update Compose documentation on website
### Is your enhancement related to a problem? Please describe

Documentation for Compose is out of date on the site as it still relates to the python implementation of podman-compose.

### Describe the solution you'd like

We are lacking some new documentation for compose since implementing new features:
- [ ] System wide install for docker compose documentation
- [ ] Indicating to use `docker-compose` from the command line
- [ ] Proper documentation on the usage of `compose.bat` with Windows
- [ ] Updated documentation with regards to future `podman compose` integration
- [ ] Examples / accessibility of using DOCKER_HOST on the command line to point to Podman before using `docker-compose`
- [ ] Updated examples on the differences in compatibility mode for windows vs mac vs linux.

We also have incorrect prompts:
- [ ] In issue https://github.com/containers/podman-desktop/issues/3535 we say to "run compose.bat" to setup DOCKER_HOST, but in the next prompt, it asks to "use compose.bat" for docker-compose. Which is not true, as `compose.bat` sets up the environment variables, then you should run the `docker-compose` binary. Until we have `podman compose` alias setup in the future. 

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### LINKED ISSUE #3615: Telling user to run compose.bat or compose script is confusing.
### Is your enhancement related to a problem? Please describe

See: https://github.com/containers/podman-desktop/issues/3557

We use the `compose` bat / bash script on Windows and Linux to set the DOCKER_HOST variables and then wrap the command around the `docker-compose` binary.

All of this is very confusing to the end user with regards to using the script as well as the upcoming implementation of `podman compose`.

### Describe the solution you'd like

We should remove references to using `compose.bat` and provide better information how to setup `DOCKER_HOST`. Either in the dialog or a link to the website.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

* Reorganized the Compose section in three pages:
  1. _Working with Compose_, with diagram and screenshot.
  2. _Setting up Compose_, with emphasis on the reference Compose engine.
  3. _Running Compose files_, with emphasis on the reference Compose engine.

### Screenshot/screencast of this PR

N/A, see website preview

### What issues does this PR fix or reference?

Closes #3615
Closes #3557
Requires #3731

### How to test this PR?

View the site

Signed-off-by: Charlie Drage <charlie@charliedrage.com>### What does this PR do?

### CODE REVIEW

**Comment by themr0c** [comment]
Ready to merge, @cdrage and @benoitf ?

File: website/docs/compose/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+1. [You set up Compose](compose/setting-up-compose).
+1. You have a [Compose file](https://github.com/compose-spec/compose-spec/blob/master/spec.md#compose-file).
+1. [You run a Compose engine on the Compose file](compose/running-compose).
+1. The Compose engine starts the containers and services, and adds a label to each resource:
+
+   - Container label: `com.docker.compose.project`
+   - Service label: `com.docker.compose.service`
+
+1. Podman Desktop detects the Compose labels, and displays the container group as a pod.
```
Comment (cdrage) [inline website/docs/compose/index.md]:
```suggestion
1. Podman Desktop detects the Compose labels, and displays the container group as a group of containers.
```

Avoid use of `pod` as a group of compose containers is just a group of containers. A pod is a different concept.

File: website/docs/compose/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+timeline
+    title Working with Compose
+    Setting up : Podman Desktop
+               : Podman
+               : Compose engine
+               : DOCKER_HOST
+    Compose : Compose file
+            : Run Compose file
+            : Add labels
+    Podman Desktop: Display pods
```
Comment (cdrage) [inline website/docs/compose/index.md]:
```suggestion
    Podman Desktop: Display multi-container applications
```

So throughout the compose documentation and spec they refer to it as "multi-container applications", we should use that: https://docs.docker.com/get-started/08_using_compose/

A group of compose containers is not called a pod.

File: website/docs/compose/running-compose.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: Running Compose files
+description: With Podman Desktop, you can manage multi-container applications defined in Compose files.
+sidebar_position: 3
+keywords: [compose]
+tags: [compose]
+---
+
+With Podman Desktop, you can manage multi-container applications defined in Compose files.
```
Comment (cdrage) [inline website/docs/compose/running-compose.md]: usually only one compose file, may flow better saying `defined in a Compose file.`?

File: website/docs/compose/running-compose.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Run the command in a terminal:
+
+  ```shell-session
+  $ docker-compose --file compose.yaml up --detach
+  ```
+
+#### Verification
+
+- The Podman Desktop **<icon icon="fa-solid fa-cube" size="lg" /> Containers** list displays the containers created by Compose grouped in a Pod with a `(compose)` suffix, such as `flask-redis (compose)`.
```
Comment (cdrage) [inline website/docs/compose/running-compose.md]: remove Pod terminology

File: website/docs/compose/setting-up-compose.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+#### Procedure
+
+1. Install the Compose engine: in the Podman Desktop status bar, click on **<icon icon="fa-solid fa-download" size="lg" />Compose**, and follow the prompts.
+1. To use the Compose engine from your terminal, [set the DOCKER_HOST variable](/docs/migrating-from-docker/using-the-docker_host-environment-variable).
+
+#### Verification
+
+1. Open a terminal.
+1. The Compose engine is in your PATH, therefore, you can display the Compose engine version:
```
Comment (cdrage) [inline website/docs/compose/setting-up-compose.md]:
```suggestion
1. The Compose reference implementation is in your PATH, therefore, you can display the Compose engine version:
```

Referring to Compose engine is fine in the other parts of the docs, but now we are talking about the CLI, I think we should be saying reference implementation.

compose-spec.io refers to it as a reference implementation as we should try to reference it as much as possible.

**Review by cdrage — CHANGES_REQUESTED** [review]

File: website/docs/compose/running-compose.md (hunk 3 above)
Comment (themr0c) [inline website/docs/compose/running-compose.md]:
```suggestion
With Podman Desktop, you can manage multi-container applications defined in a Compose file.
```

File: website/docs/compose/running-compose.md (hunk 4 above)
Comment (themr0c) [reply website/docs/compose/running-compose.md]:
```suggestion
- The Podman Desktop **<icon icon="fa-solid fa-cube" size="lg" /> Containers** list displays the containers created by Compose grouped in a container group with a `(compose)` suffix, such as `flask-redis (compose)`.
```

**Comment by themr0c** [comment]
All suggestions applied, can you have another look @cdrage ?

**Comment by cdrage** [comment]
Uh oh. Mermaid is not working.

The screenshot is here:
Screenshot 2023-09-06 at 11 15 41 AM


According to the issue: https://github.com/facebook/docusaurus/issues/9032 this is being worked on and planned for the next release (3.0).

So for the time being, can you please take a screenshot of the diagram + add this to the PR?

We should open up an issue for the time being to re-enable mermaid js when docusaurus has a fix.

**Review by cdrage — DISMISSED** [review]

**Comment by cdrage** [comment]
LGTM! Thank you for these changes.

File: website/docs/compose/setting-up-compose.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+tags: [compose]
+---
+
+# Setting up Compose
+
+Podman Desktop can install the Compose engine.
+
+#### Procedure
+
+1. Install the Compose engine: in the Podman Desktop status bar, click on **<icon icon="fa-solid fa-download" size="lg" />Compose**, and follow the prompts.
```
Comment (themr0c) [inline website/docs/compose/setting-up-compose.md]:
```suggestion
1. Install the Compose engine: in the Podman Desktop status bar, click **<icon icon="fa-solid fa-download" size="lg" />Compose**, and follow the prompts.
```

**Comment by themr0c** [comment]
mermaid not working on your environment: maybe it is not installed yet on your local environment? Can you double-check by running `yarn install` in the `website` directory?

**Comment by cdrage** [comment]
> mermaid not working on your environment: maybe it is not installed yet on your local environment? Can you double-check by running `yarn install` in the `website` directory?

it works on my environment but not on netlify, and I'm afraid it'll probably fail when we push it to production as well on github pages.

what we COULD do it possibly merge this in and see what happens. just netlify will not be working for now.

**Comment by benoitf** [comment]
@themr0c 
you need to check the 'build' mode and not the 'dev' mode

in website folder do

`yarn build`

then

`yarn serve`

**Comment by themr0c** [comment]
@cdrage @benoitf Using a rendered version of the diagram. Ready to approve and merge.

**Review by cdrage — APPROVED** [review]
LGTM! Docs look great and thank you for being patient with the back-and-forth on reviews and changes.

The link to the actual mermaid render also looks wonderful.

Thanks again!

**Pull request merged**